crates/storage_impl/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "storage_impl"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Please add a README and more fields present in our other crates.

https://github.com/juspay/hyperswitch/blob/c464cc510ded595ea846e7da95f60919614e2bd3/crates/common_enums/Cargo.toml#L1-L8